salt/modules/gnomedesktop.py

```diff
@@ -50,6 +52,17 @@ def __init__(self, user, schema, key):
         self.UID = None
         self.HOME = None
 
+    @property
+    def gestting_command(self):
```

Comment:
I'm guessing this was supposed to be `gsetting_command`, not `gestting_command`.